Apps/Sandcastle/gallery/Resolution Scaling.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Set a custom resolution scale or enable device pixel resolution.">
+    <meta name="cesium-sandcastle-labels" content="Post Processing">
```

Comment:
Put this in the `Showcases` category instead since it's not strictly post-processing related.